v3/plugins/redhat/vscode-camelk/0.0.14/meta.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+publisher: redhat
+name: vscode-camelk
+version: 0.0.14
+type: VS Code extension
+displayName: Tooling for Apache Camel K
+title: Tooling for Apache Camel K
+description: Support for Apache Camel K functionality
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/camel-tooling/vscode-camelk
+category: Language
+firstPublicationDate: '2020-06-18'
+spec:
+  containers:
+  - image: "quay.io/eclipse/che-sidecar-camelk:0.0.14-59d258b"
+    name: vscode-camelk
+    memoryLimit: "1G"
+  extensions:
+  - https://download.jboss.org/jbosstools/vscode/stable/vscode-camelk/vscode-camelk-0.0.14-622.vsix
+  - https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-kubernetes-tools/vscode-kubernetes-tools-1.2.0.vsix
+  - https://github.com/redhat-developer/vscode-yaml/releases/download/0.8.0/redhat.vscode-yaml-0.8.0.vsix
```

Comment:
Can you replace it with https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-yaml/vscode-yaml-0.8.0.vsix